smart-contracts/contracts-common/concordium-contracts-common/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased changes
+- Fixed `Display` trait on `VersionedModuleSchema` when module contained multiple contracts to render all of them.
```

Comment:
This is a nitpick, but the style in the changelog is to use present tense, not past, so "Fix ...".

This is also the style we try to use in commit messages.